src/main/java/seedu/address/logic/parser/RecipeBookSyntax.java

```diff
@@ -15,4 +15,6 @@
 
     /** Used to indicate index in a command */
     public static final Prefix PREFIX_INDEX = new Prefix("-x ");
+
+    public static final String SEPARATOR_SYMBOL = "  ";
```

Comment:
This change should allow separating values by the pipe symbol.
```suggestion
    public static final String SEPARATOR_SYMBOL = "\\|";
```

Reply:
I agree with this, looks good!